lua/entities/prop_trash.lua

```diff
@@ -167,6 +169,9 @@ if CLIENT then
 			end
 		end
 	end
+	if IsValid((self.UseTable or {})[1]) then
```

Comment:
did you add this?

Reply:
Yes, it's to make people exit the chairs while in theaters because people keep on blocking screens by sitting in a chair

Reply:
have you considered any negative side effects of that?

Reply:
yes I just realized I forgot to add the check to make sure it's in a theater...

Reply:
The code is totally wrong in other ways. I mean the potential side effects of what you're actually TRYING to do

Reply:
@GrateBrittan

Reply:
Yes I read it

Reply:
You don't see any issue with preventing people from sitting in chairs in theaters?

Reply:
yes

Reply:
first, this is wrapped in `if CLIENT` and ExitVehicle is serverside only
second, it's not in the Think function so it would only run once upon initialization
third, where does UseTable ever get defined
fourth, who cares if someone sitting can block the screen, just use fullscreen

Reply:
Everybody told me to fix it